openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,680 @@
+                  isSummaryPanelOpen={Boolean(previewAsset)}
+                  permissions={MOCK_TAG_PERMISSIONS}
+                  ref={assetTabRef}
+                  type={AssetsOfEntity.TAG}
+                  onAddAsset={() => setAssetModalVisible(true)}
+                  onAssetClick={handleAssetClick}
+                  onRemoveAsset={handleAssetSave}
+                />
+              ),
+              ...COMMON_RESIZABLE_PANEL_CONFIG.LEFT_PANEL,
+            }}
+            hideSecondPanel={!previewAsset}
+            secondPanel={{
+              children: previewAsset && (
+                <EntitySummaryPanel
+                  entityDetails={previewAsset}
+                  handleClosePanel={() => setPreviewAsset(undefined)}
+                />
+              ),
+              ...COMMON_RESIZABLE_PANEL_CONFIG.RIGHT_PANEL,
+              className:
+                'entity-resizable-right-panel-container tag-resizable-panel-container',
+            }}
+          />
+        ),
+      },
+    ];
+
+    return items;
+  }, [
+    tagItem,
+    previewAsset,
+    activeTab,
+    assetCount,
+    assetTabRef,
+    handleAssetSave,
+    isDescriptionEditable,
+    editTagsPermission,
+    editDescriptionPermission,
+  ]);
+  const icon = useMemo(() => {
+    if (tagItem?.style?.iconURL) {
+      return (
+        <img
+          alt={tagItem.name ?? 'Tag Icon'}
+          className="align-middle object-contain"
+          data-testid="icon"
+          height={36}
+          src={tagItem.style?.iconURL}
+          width={32}
+        />
+      );
+    }
+
+    return <IconTag className="h-9" style={{ color: DE_ACTIVE_COLOR }} />;
+  }, [tagItem]);
+
+  useEffect(() => {
+    getTagData();
+    fetchClassificationTagAssets();
+  }, []);
+
+  useEffect(() => {
+    if (tagItem) {
+      fetchCurrentTagPermission();
+    }
+  }, [tagItem]);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (!tagItem) {
+    return null;
+  }
+
+  return (
+    <PageLayoutV1 pageTitle={tagItem.name}>
+      <Row gutter={[0, 8]}>
+        <Col span={24}>
+          <Row
+            className="data-classification"
+            data-testid="data-classification"
+            gutter={[0, 12]}>
+            <Col className="p-x-md" flex="auto">
+              <EntityHeader
+                badge={
+                  !editTagsPermission && (
+                    <Space>
+                      <Divider className="m-x-xs h-6" type="vertical" />
+                      <StatusBadge
+                        dataTestId="disabled"
+                        label="Disabled"
+                        status={StatusType.Stopped}
+                      />
+                    </Space>
+                  )
+                }
+                breadcrumb={breadcrumb}
+                entityData={tagItem}
+                entityType={EntityType.TAG}
+                icon={icon}
+                serviceName=""
+                titleColor={tagItem.style?.color ?? 'black'}
```

Comment:
Why we are directly using `black', i think we should use Hex from constant

Reply:
Updated.